Source/WebCore/PAL/pal/spi/cg/CoreGraphicsSPI.h

```diff
@@ -124,18 +124,6 @@ typedef struct CGSRegionObject* CGSRegionObj;
 typedef struct CGSRegionObject* CGRegionRef;
 #endif
 
-#ifdef CGFLOAT_IS_DOUBLE
```

Comment:
I don't think we should remove these, nor should we replace the existing use of them.

Reply:
Resolved!